scripts/west_commands/runners/openocd.py

```diff
@@ -12,6 +12,11 @@
 from os import path
 from pathlib import Path
 from zephyr_ext_common import ZEPHYR_BASE
+import socket
+import selectors
+import sys
+import time
+import shutil
```

Comment:
Nit: maybe put the import in alphabetical order as you're adding a bunch of new ones.